### ISSUE TITLE
Implementing the "You Win" title

### ISSUE THREAD

**Issue by SashoStefanov** [SashoStefanov/-Elinarinla-Teamwork#6]
Must implement the You Win title, so when user wins the title shall occur.

**Comment by MargaritaSv** [comment]
Сега ще го направя надписа.
А някои същност направил ли ги е вече?

**Comment by SashoStefanov** [comment]
Да, аз го напрвих, но не е имплементирано в кода. You Lost и Hangman Title също ги има, но трябва да се имплементират. Май бяхме говорили Стефан да ги вкара в кода, но май нищо не е качил още.
